crates/plugins/src/manager.rs

```diff
@@ -364,6 +374,15 @@ fn verify_checksum(plugin_file: &Path, expected_sha256: &str) -> Result<()> {
     }
 }
 
+fn maybe_get_auth_header() -> Result<HeaderMap> {
```

Comment:
```suggestion
/// Get the request headers for a call to the plugin API
///
/// If set, this will include the user provided authorization header.
fn request_headers() -> Result<HeaderMap> {
```